package.json

```diff
@@ -95,13 +95,13 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.0-9"
   },
   "dependencies": {
-    "@angular/common": "^5.0.0",
-    "@angular/compiler": "^5.0.0",
-    "@angular/core": "^5.0.0",
-    "@angular/platform-browser": "^5.0.0",
-    "@angular/platform-browser-dynamic": "^5.0.0",
+    "@angular/common": "^6.0.0",
+    "@angular/compiler": "^6.0.0",
+    "@angular/core": "^6.0.0",
+    "@angular/platform-browser": "^6.0.0",
+    "@angular/platform-browser-dynamic": "^6.0.0",
     "core-js": "^2.4.1",
-    "rxjs": "^5.5.2",
+    "rxjs": "^6.1.0",
     "zone.js": "^0.8.14"
```

Comment:
Can you align zone.js [version](https://github.com/angular/angular/blob/master/packages/core/package.json#L20) with `@angular/core` as well, please?